charts/fybrik/templates/manager-deployment.yaml

```diff
@@ -96,6 +96,20 @@ spec:
                 name: razee-credentials
             {{- end }}
           env:
+            - name: TLS_ENABLED
+              value: {{ .Values.global.tls.enabled | quote | toString }}
+            {{- if .Values.global.tls.enabled }}
+            - name: MTLS_ENABLED
+              value: {{ .Values.global.tls.mtlsEnabled | quote | toString }}
+            - name: CERT_SECRET_NAME
+              value: {{ .Values.manager.tls.certs.clientCertSecretName }}
+            - name: CERT_SECRET_NAMESPACE
+              value: {{ .Values.manager.tls.certs.clientCertSecretNamespace }}
+            - name: CACERT_SECRET_NAME
+              value: {{ .Values.manager.tls.certs.cacertSecretName }}
+            - name: CACERT_SECRET_NAMESPACE
+              value: {{ .Values.manager.tls.certs.cacertSecretNamespace }}
```

Comment:
Where are these env vars used in the manager code?

Reply:
in the auto-genrated code pkg/connectors/datacatalog/openapiclient/client.go and pkg/connectors/policymanager/openapiclient/client.go. The latest version of openapi generator tool [supports](https://github.com/OpenAPITools/openapi-generator#11---compatibility) OpenAPI Spec 3.0 and it does not contain TLS support so I had to insert the tls support in the generated code.